integration/utils/contLoadGenerator.go

```diff
@@ -422,7 +422,7 @@ func (lg *ContLoadGenerator) sendTokenTransferTx(workerID int) {
 	lg.log.Trace().Msgf("creating transfer script")
 	transferScript, err := TokenTransferScript(
 		lg.fungibleTokenAddress,
-		acc.address,
+		lg.flowTokenAddress,
```

Comment:
are you sure about this?

Reply:
this will change the behaviour of nodes transferring money to each other to flow token account transferring flow to accounts.

Reply:
I just tested, it works when I switch back to use `acc.account`. 

And after a while, it log shows the following error, I guess it means the execution node has reached to its limit:
```
11:22AM INF creating and funding 100 accounts...
11:22AM WRN error getting tx result, will put tx back to queue after sleep error="client: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp: lookup localhost: no such host\"" tx_id=a6be2f52a30de4048fc27eb5e299ddec34ac5202ecbef46c70028ed925f9520f worker_id=20
11:22AM WRN error getting tx result, will put tx back to queue after sleep error="client: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp: lookup localhost: no such host\"" tx_id=a6be2f52a30de4048fc27eb5e299ddec34ac5202ecbef46c70028ed925f9520f worker_id=22
```